Base/res/ladybird/about-pages/settings.html

```diff
@@ -990,6 +1045,56 @@ <h3 id="site-settings-title" class="dialog-title"></h3>
                 });
             });
 
+            // DNS settings
+            const loadDnsSettings = settings => {
+                dnsUpstream.value = settings.mode;
+
+                if (settings.mode === "custom") {
+                    dnsServer.value = settings.server;
+                    dnsPort.value = settings.port;
+                    dnsType.value = settings.type;
+                    customDnsSettings.classList.remove("hidden");
+                } else {
+                    dnsServer.value = "";
+                    dnsType.value = "udp";
+                    dnsPort.value = "53";
+                    customDnsSettings.classList.add("hidden");
+                }
+
+                if (settings.forciblyEnabled) {
```

Comment:
Not a huge deal, but there is now a difference of behavior when the command line is used for DNS settings vs. autoplay. When autoplay is overridden via the command line, we hide all of its settings, to avoid weird interactions between the cmd line and persisted settings. Might make sense to do that here too. Up to you.

Reply:
I can make them noninteractive somehow, would that be better?
I'd rather show the settings' values personally

Reply:
Yeah disabling the inputs should be fine too. Mostly just want to avoid weird interactions between transient command line settings and these persisted settings. But the command line is largely for testing, so whatever's simplest is fine.